internal/libs/alpnmux/mux.go

```diff
@@ -110,13 +113,15 @@ func (l *ALPNMux) RegisterProto(proto string, tlsConf *tls.Config) (net.Listener
 	}
 
 	if l.log != nil && l.log.IsDebug() {
-		l.log.Debug("registered", "proto", proto)
+		event.WriteSysEvent(ctx, op, map[string]interface{}{"msg": "registered", "proto": proto})
```

Comment:
Similar question here, how is the conditional above impacted by there being no log levels?

Reply:
Here is another instance of where the only current signal is coming from an embedded logger in the receiver so at this point there's not much to do but allow it to provide a signal